src/Moq/Mock.cs

```diff
@@ -407,24 +407,25 @@ internal void AddFixedReturnValueSetup(MethodInfo method, IReadOnlyList<Expressi
 			this.Setups.Add(new FixedReturnValueSetup(method, arguments, expression, returnValue));
 		}
 
-		internal void AddFixedReturnValueSetup(MethodInfo method, object returnValue)
+		internal void AddFixedReturnValueSetup(MethodInfo method, object[] arguments, object returnValue)
```

Comment:
(Hmm. Which is better, having a setup where an original `It.IsAny<TArg>` (turned `default(TArg)` in the invocation's `arguments` array) is treated as `default(TArg)`; or having a setup where any original arguments like `1` end up being misrepresented as `It.IsAny<int>`?)